skimage/transform/_geometric.py

```diff
@@ -248,6 +300,9 @@ def estimate(self, src, dst):
         H.flat[list(self._coeffs) + [8]] = - V[-1, :-1] / V[-1, -1]
         H[2, 2] = 1
 
+        # De-center and de-normalize
+        H = np.dot(np.linalg.inv(dst_matrix), np.dot(H, src_matrix))
```

Comment:
Could be done, of course. I prefer keeping it the way it is now, unless you have a strong preference...